base/linalg/uniformscaling.jl

```diff
@@ -178,6 +178,11 @@ function copy!(A::AbstractMatrix, J::UniformScaling)
     return A
 end
 
+function cond{T}(J::UniformScaling{T})
+    onereal = inv(one(real(J.λ)))
```

Comment:
It is for `Float64`, `Float32`, and machine integers so I don't think it is a problem.